client/my-sites/stats/stats-period-navigation/index.jsx

```diff
@@ -55,13 +61,17 @@ class StatsPeriodNavigation extends PureComponent {
 
 		return (
 			<div className="stats-period-navigation">
-				<a
-					className="stats-period-navigation__previous"
-					href={ `${ url }?startDate=${ previousDay }` }
-					onClick={ this.handleClickPrevious }
-				>
-					<Gridicon icon="arrow-left" size={ 18 } />
-				</a>
+				{
+					<a
+						className={ classNames( 'stats-period-navigation__previous', {
+							'is-disabled': hidePreviousArrow,
```

Comment:
Did you consider removing the arrow rather than hiding it with a class?

Reply:
Yes. I initially removed it completely (it's faster to code just that :D) but the component was hiding the other arrow with a class so to be consistent, I went with the class method.